services/console-proxy/server/src/com/cloud/consoleproxy/ConsoleProxyAjaxHandler.java

```diff
@@ -20,179 +20,26 @@
 
 import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.net.URLDecoder;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
-import com.sun.net.httpserver.Headers;
-import com.sun.net.httpserver.HttpExchange;
-import com.sun.net.httpserver.HttpHandler;
-
 import com.cloud.consoleproxy.util.Logger;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
-public class ConsoleProxyAjaxHandler implements HttpHandler {
+public class ConsoleProxyAjaxHandler extends AbstractHandler {
     private static final Logger s_logger = Logger.getLogger(ConsoleProxyAjaxHandler.class);
 
     public ConsoleProxyAjaxHandler() {
     }
 
-    @Override
-    public void handle(HttpExchange t) throws IOException {
```

Comment:
It seems that the functions `handle()` and `dohandle()` are removed here and added down again, technically, not a lot changed, can you make sure that the diff is more accurate?